bin/webpack.js

```diff
@@ -1,45 +1,25 @@
-#!/usr/bin/env node
-
 /*
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
 var path = require('path');
-// var fs = require('fs');
-// Local version replace global one
-
-process.title = 'webpack';
 
+// Local version replace global one
 try {
-	var localWebpack = require.resolve(path.join(process.cwd(), 'node_modules', 'webpack-cli', 'bin', 'webpack.js'));
-	if(localWebpack && path.relative(localWebpack, __filename) !== '') {
+	var localWebpack = require.resolve(path.join(process.cwd(), 'node_modules',
+	'webpack', 'node_modules', 'webpack-cli', 'bin', 'webpack.js'));
+	if(__filename !== localWebpack) {
```

Comment:
Added in new commit, thanks for the heads-up! A bit hard to test this in dev, but used `resolveCwd.silent`, thanks for an excellent api!